plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx

```diff
@@ -72,7 +73,17 @@ const copyToClipboard = ({ about }: { about: DevToolsInfo | undefined }) => {
 /** @public */
 export const InfoContent = () => {
   const classes = useStyles();
+  const [resources, setResources] = useState<string | undefined>(undefined);
   const { about, loading, error } = useInfo();
+  useSignalApi('devtools:resources', message => {
+    setResources(message.resources as string);
+  });
+
+  useEffect(() => {
+    if (!loading && !error && about) {
+      setResources(about.resourceUtilization);
```

Comment:
I think it's enough to have this one just depend on `resourceUtilization` and set it whenever it changes and isn't undefined?

Reply:
I think we can drop the devtools part for now. It was mostly to showcase some real example of signals usage.